bulk-import-tools/internal/orchcli/orchcli.go

```diff
@@ -243,6 +246,56 @@ func (oC *OrchCli) InstanceExists(ctx context.Context, sn, uuid string) (bool, e
 	return false, nil
 }
 
+func (oC *OrchCli) GetHostID(ctx context.Context, sn, uuid string) (string, error) {
+	if uuid == "" {
+		return "", e.NewCustomError(e.ErrInternal)
+	}
+
+	uParsed := *oC.SvcURL
+	uParsed.Path = path.Join(uParsed.Path, fmt.Sprintf("/v1/projects/%s/compute/hosts", oC.Project))
+	query := uParsed.Query()
+	query.Set("filter", fmt.Sprintf("%s=%q", "uuid", uuid))
```

Comment:
I don't fully get why we only filter on UUID? Should we build the filter so we filter on both SN and UUID if both are provided?

Reply:
Agree. Had undone an earlier filter logic but was missing one case (only SN in pre-registered host & 412 received). Thanks. Simplified with "sn AND uuid" filter now with at least one non-empty.

Reply:
Addressed in https://github.com/open-edge-platform/infra-core/pull/97#discussion_r2073754190